### PR TITLE
ADMINUI-2251 Network details page - Restore hyperlinks for VM/server …

### PR DESCRIPTION
**Pull request by Arc311**
…uuid shown with IP addresses

### CODE REVIEW

File: www/js/components/pages/network/addresses-list.jsx [hunk 1]
```diff
@@ -236,11 +236,38 @@ var AddressesList = React.createClass({
...
+
+        var navigateToAdresses = function (address, event) {
+            if (event.metaKey || event.ctrlKey) {
+                return;
+            }
+            event.preventDefault();
+
+            var type = address.belongs_to_type;
+            if (type === 'zone') {
+                adminui.router.showVm(address.belongs_to_uuid);
```
Comment (YZaicev) [inline www/js/components/pages/network/addresses-list.jsx]: address.belongs_to_uuid to var

File: www/js/components/pages/network/addresses-list.jsx [hunk 2]
```diff
@@ -236,11 +236,38 @@ var AddressesList = React.createClass({
...
+            }
+            return prefix ? _.str.sprintf('/%s/%s', prefix, address.belongs_to_uuid) : '';
+        };
+
+        var navigateToAdresses = function (address, event) {
+            if (event.metaKey || event.ctrlKey) {
+                return;
+            }
+            event.preventDefault();
+
```
Comment (YZaicev) [inline www/js/components/pages/network/addresses-list.jsx]: if (type === 'zone' || type === 'server') {
  adminui.router[type === 'zone' ? 'showVm' : 'showServer'](address.belongs_to_uuid);
}

File: www/js/components/pages/network/addresses-list.jsx [hunk 3]
```diff
@@ -236,11 +236,38 @@ var AddressesList = React.createClass({
     render: function () {
         var state = this.state.state;
         var self = this;
+
+        var getAdressesUrl = function (address) {
+            var type = address.belongs_to_type;
+            var prefix;
+            if (type === 'zone') {
```
Comment (YZaicev) [inline www/js/components/pages/network/addresses-list.jsx]:
```
if (type === 'zone' || type === 'server') {
  return _.str.sprintf('/%s/%s', type === 'vms' ? 'servers' : 'showServer', address.belongs_to_uuid);
}
```

**Comment by YZaicev** [comment]
:1234:

**Comment by askfongjojo** [comment]
Merged as c8fb8b3

**Pull request closed without merging**